### PR TITLE
Temperature

### PR DESCRIPTION
**Pull request by rusefillc**

### CODE REVIEW

File: firmware/interpolation.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#include "interpolation.h"
+
+int interpolateInt(int x1, int y1, int x2, int y2, int x)
```
Comment (mck1117) [inline firmware/interpolation.cpp:3]: Can we just use floats? Float performance on ARM is pretty good even for soft float.

**Comment by rusefillc** [comment]
@dron0gus

**Comment by dron0gus** [comment]
https://github.com/rusefi/wideband/pull/4/commits/080b8ba009ab5a7ba11b6507a39b8e18f57fdb02

**Comment by mck1117** [comment]
@dron0gus and @rusefillc, why are you guys using whatever ridiculous workflow involves @dron0gus making commits and @rusefillc making PRs?

**Comment by dron0gus** [comment]
Long story.
I'm doing bring up.
And my draft PR is just an indication of progress.

**Comment by rusefillc** [comment]
i am a believer in smaller manageable PRs

as long as the draft is 80% on point I believe there is value in merging pieces of it. At the moment we believe that F103 is functional but no clear plan on real vehicle testing only unclear plans :) still my preference is to move codebase forward as long as we can identify not insane pieces

**Comment by mck1117** [comment]
I'm going to let you guys resolve this vs. #58 which appear to be near duplicates before I merge either.

**Comment by rusefillc** [comment]
@mck1117 I assume that #58 is a prototype of this?

@dron0gus shall we close #58 and merge this #63?

**Comment by rusefillc** [comment]
@mck1117 we now only have this one open please revisit this one

**Pull request merged**